canonical/store-components
# Create card component

## Done
Added the initial `PackageCard` component

## QA
- Go to https://store-components-6.demos.haus/?path=/story/packagecard--default-story
- Check that there is a `PackageCard` component

## Issue
Fixes https://warthogs.atlassian.net/browse/WD-1622
